docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -27,6 +27,46 @@ kubectl label namespace default knative-eventing-injection=enabled
    with the appropriate permissions, you need to modify the
```

Comment:
Global: Let's add lines before these large .yaml files that instruct folks to copy the .yaml below into a file named `actualFileName.yaml`, modifying the file as necessary.

You can look at this page for a model of this: https://knative.dev/docs/serving/samples/hello-world/helloworld-php/

Reply:
Oh, don't know if you've seen "global" before, but in case you haven't, it just means "throughout this file" or "everywhere."